website/pages/themes.vue

```diff
@@ -0,0 +1,284 @@
+          : 1
+      })
+      return categories
+    },
+    categoryArray() {
+      return themes.reduce((acc, theme) => {
+        acc = [...acc, ...theme.categories]
+        return acc
+      }, [])
+    },
+    filteredThemes() {
+      if (this.filters.length !== 0) {
+        return this.themes.filter(theme =>
+          theme.categories.some(cat => {
+            return this.filters.includes(cat)
+          })
+        )
+      } else {
+        return this.themes
+      }
+    }
+  },
+  methods: {
+    formatCategories(categories) {
+      return categories.join(', ')
+    },
+    getThumbnail(name) {
+      return require('../images/themes/' + name)
+    },
+    toggleSelection(key) {
+      if (this.filters.includes(key)) {
+        this.filters.splice(this.filters.indexOf(key), 1)
+      } else {
+        this.filters.push(key)
+      }
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.themes {
+  &__container {
+    display: flex;
+  }
+
+  &__categories {
+    padding-right: 1rem;
+    flex-basis: var(--sidebar-width);
+
+    &--header {
+      text-transform: uppercase;
+      padding-bottom: 1rem;
+      margin-bottom: 1rem;
+      border-bottom: 1px solid var(--border-color);
+    }
+
+    &--list {
+      &-item {
+        display: flex;
+        justify-content: space-between;
+        text-transform: capitalize;
+      }
+    }
+  }
+
+  &__gallery {
+    padding-left: 1rem;
+    width: 100%;
+
+    &--header {
+      padding-bottom: 1rem;
+      margin-bottom: 1rem;
+      border-bottom: 1px solid var(--border-color);
+
+      span {
+        font-weight: 600;
+      }
+    }
+
+    &--list {
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      grid-gap: 3rem;
+    }
+  }
+
+  &__card {
+    &--header:hover &--thumbnail {
+      box-shadow: 0px 4px 8px rgba(46, 41, 51, 0.08),
+        0px 8px 16px rgba(71, 63, 79, 0.16);
+    }
+
+    &--thumbnail {
+      height: 12rem;
+      background-size: cover;
+      background-position: center;
+      transition: 0.3s;
+      margin-bottom: 1rem;
+    }
+
+    &--title {
+      font-weight: 600;
+    }
+
+    &--footer {
+      display: flex;
+      justify-content: space-between;
+      color: hsla(270, 3.674150944000001%, 0%, 0.36);
+    }
+
+    &--subtitle {
+      text-transform: capitalize;
+      font-size: 0.8rem;
+    }
+
+    &--ext {
+      svg {
+        fill: hsla(270, 3.674150944000001%, 0%, 0.36);
+      }
+
+      svg:hover {
+        fill: black;
+      }
+    }
+  }
+
+  @media (max-width: 960px) {
+    &__gallery {
+      &--list {
+        grid-template-columns: repeat(2, 1fr);
+      }
+    }
+  }
+
+  @media (max-width: 768px) {
+    &__categories {
+      display: none;
+    }
+  }
+
+  @media (max-width: 660px) {
+    &__gallery {
+      &--list {
+        grid-template-columns: 1fr;
+        justify-items: center;
+      }
+    }
+
+    &__card {
+      width: 20rem;
+    }
+  }
+}
+</style>
```

Comment:
let's use PostCSS only 😂

Reply:
oh lol i didnt know there is no sass-loader xD